definitions/infra-awsvpcnetworkinterface/definition.yml

```diff
@@ -1,2 +1,5 @@
 domain: INFRA
 type: AWSVPCNETWORKINTERFACE
+configuration:
+  entityExpirationTime: DAILY
+  alertable: true
```

Comment:
Reminder: this will require NerdGraph to enable the type as alertable (we haven't automated this yet).